tests/catch_up_subscription_handles_small_batch_sizes.php

```diff
@@ -59,6 +59,13 @@ protected function setUpAsync(): Promise
         });
     }
 
+    protected function tearDownAsync(): Promise
+    {
+        $this->connection->close();
+
+        return parent::tearDownAsync();
+    }
```

Comment:
What's the reason why you're not extending `EventStoreConnectionTestCase` here?

Reply:
changed